lib/views/pages/account_settings/account_settings.dart

```diff
@@ -26,8 +26,9 @@ class AccountSettings extends HookConsumerWidget {
           margin: const EdgeInsets.only(top: 40.0),
           decoration: new BoxDecoration(
```

Comment:
replace new with const keyword for BoxDecoration ([reference](https://dart-lang.github.io/linter/lints/prefer_const_constructors.html))